Description: Disable smooth scrolling
 Comment out code enabling smooth scrolling disregarding preferences
 to leave decision whenever use the smooth scrolling or not to user.
Author: Thomas Richter <richter@rus.uni-stuttgart.de>
Reviewed-by: Dmitry Smirnov <onlyjob@member.fsf.org>
Origin: other, https://bugs.debian.org/624095
Bug-Debian: https://bugs.debian.org/624095
Last-Update: 2020-06-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/text/fmt/xp/fv_View.cpp
+++ b/src/text/fmt/xp/fv_View.cpp
@@ -14363,6 +14363,8 @@
 	{
 		m_iNumHorizPages = m_pLayout->countPages();
 	}
+/*	Debian bug #624095: smooth scrolling enabled, preferences ignored.
+ *	Commenting to leave decision whenever use the smooth scrolling to user.
 	if (m_iNumHorizPages > 1)
 	{
 		XAP_App::getApp()->setEnableSmoothScrolling(false);
@@ -14371,6 +14373,7 @@
 	{
 		XAP_App::getApp()->setEnableSmoothScrolling(true);
 	}
+*/
 	if(iOldNo != static_cast<UT_sint32>(m_iNumHorizPages))
 	{
 		UT_uint32 iPrevYOffset = m_yScrollOffset;
